topics/ecology/tutorials/PAMPA-toolsuite-tutorial/tutorial.md

```diff
@@ -0,0 +1,847 @@
+>    - {% icon param-select %} *"using column"*: `Column: 2`
+>    - {% icon param-repeat %} Click *"+ Insert Check"*: 
+>         - {% icon param-text %} *"Find Regex"*: `([0-9]{4})_` 
+>         - {% icon param-text %} *"Replacement"*: `\1`
+> 
+>    > ### {% icon question %} Question
+>    > What did we do with this tool?
+>    > > ### {% icon solution %} Solution
+>    > > We removed the `_` we added in the first part of this "Hands_on" from the values of the second column 
+>    > > ```year``` to get proper numeric values back.
+>    > {: .solution}
+>    {: .question}
+>
+> 4. **Cut columns from a table (cut)** {% icon tool %} with following parameters :
+>    - {% icon param-files %} *"File to cut"*: `#Concatenate` Column Regex Find And Replace data file
+>    - {% icon param-select %} *"Operation"*: `Keep`
+>    - {% icon param-select %} *"Delimited by"*: `Tab`
+>    - {% icon param-select %} *"Cut by"*: `fields`
+>      - {% icon param-select %} *"List of Fields"*: `Column: 1` `Column: 2` `Column: 3` `Column: 4` `Column: 9` `Column: 10`
+>
+>    Check if the output `#Concatenate` Advanced Cut data file has the following column names ```"Survey","year","Quarter",
+>    "Area","location","yearlocation"```.
+>
+> 5. **Sort data in ascending or descending order** {% icon tool %} in the "Text Manipulation" 
+>    tool collection with following parameters :
+>    - {% icon param-files %} *"Sort Query"*: `#Concatenate` Advanced Cut data file
+>    - {% icon param-text %} *"Number of header lines"*: `1`
+>    - *"1: Column selections"*
+>      - {% icon param-select %} *"on column"*: `Column: 6`
+>      - {% icon param-check %} *"in"*: `Ascending order`
+>      - {% icon param-check %} *"Flavor"*: `Alphabetical sort`
+>      - {% icon param-select %} *"Output unique values"*: `Yes`
+>      - {% icon param-select %} *"Ignore case"*: `No`
+>
+>    The output `#Concatenate` Sort data file must have fewer lines than the `#Concatenate` Advanced Cut data file.
+>
+> 6. **Regex Find And Replace** {% icon tool %} with following parameters :
+>    - {% icon param-files %} *"Select lines from"*: `#Concatenate` Sort data file
+>    - {% icon param-repeat %} Click *"+ Insert Check"*: 
+>         - {% icon param-text %} *"Find Regex"*: `yearlocation` 
+>         - {% icon param-text %} *"Replacement"*: `observation.unit`
+>    - {% icon param-repeat %} Click *"+ Insert Check"*: 
+>         - {% icon param-text %} *"Find Regex"*: `location` 
+>         - {% icon param-text %} *"Replacement"*: `site`
+>
+>    Here, we define ```site``` represent the same geographical level as ```location```.
+>
+> 7. Tag the output `#Concatenate` Regex Find And Replace data file with `#unitobs`
+>
+>    Check if the output `#Concatenate #unitobs` data file has the following column names ```"Survey","year","Quarter",
+>    "Area","site","observation.unit"```.
+>
+{: .hands_on}
+
+# Step 2 & 3: GLMMs and plots on ```Community``` and ```Species-Population``` metrics
```

Comment:
```suggestion
# GLMMs and plots on "Community" and "Species-Population" metrics
```

Reply:
I don't think the use of the three backticks is necessary in many cases in this tutorial, I would remove them. Additionally from the list below ("or ```Shannon index```" should probably just be "Shannon index")